### PR TITLE
replace TextView to EmojiTextView

### LINKED ISSUE #332: [Assigned] Use EmojiTextView for item_feed.xml
## Overview (Required)
- Title says everything

### PR DESCRIPTION
**Pull request by hisakioomae**
## Issue
- close #332 

## Overview (Required)
- replace TextView to EmojiTextView 

## Links
- https://developer.android.com/reference/android/support/text/emoji/widget/EmojiTextView.html

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/res/layout/item_feed.xml [hunk 1]
```diff
@@ -15,20 +15,21 @@
         android:layout_height="wrap_content"
         android:background="?attr/selectableItemBackground"
         >
-        <TextView
+        <android.support.text.emoji.widget.EmojiTextView
```
Comment (takahirom) [inline app/src/main/res/layout/item_feed.xml]: Thanks!  👍 
Please use TextView for this 🙇

Comment (hisakioomae) [reply app/src/main/res/layout/item_feed.xml]: OK!
I will try it. :smile: 
Because the date does not use emoji, right? :worried:

Comment (takahirom) [reply app/src/main/res/layout/item_feed.xml]: Yes 👍 That's right 😄

**Comment by takahirom** [comment]
LGTM 👍 Thank you for contributing 😄

**Comment by hisakioomae** [comment]
🙇

**Pull request merged**